node_exporter/node_exporter.spec

```diff
@@ -29,20 +29,20 @@ Prometheus exporter for machine metrics, written in Go with pluggable metric col
 
 %install
 mkdir -vp %{buildroot}%{_sharedstatedir}/prometheus
-install -D -m 755 %{name} %{buildroot}%{_bindir}/%{name}
+install -D -m 755 %{name} %{buildroot}/%{_bindir}/%{name}
```

Comment:
The `/` between buildroot and bindir is not needed since bindir starts with a `/`